script.js

```diff
@@ -0,0 +1,76 @@
+const booksListContainer = document.querySelector(".books-list");
+const form = document.querySelector(".form");
+
+let books = [];
+if (localStorage.getItem("books")) {
+  books = JSON.parse(localStorage.getItem("books"));
+}
+
+function loadItems(booksArr) {
+  const booksList = booksArr.map(
+    (bk) => `<li class="book">
+<p>${bk.title}</p>
+ <p>${bk.author}</p>
+ <button class="btn btn-remove" data-id="${bk.id}">Remove</button>
+<hr />
+</li>`,
+  );
+
+  booksListContainer.innerHTML = booksList;
+}
+
+loadItems(books);
+
+function appendBook(bookObj) {
+  const listItem = `<li class="book">
+<p>${bookObj.title}</p>
+ <p>${bookObj.author}</p>
+ <button class="btn btn-remove" data-id="${bookObj.id}">Remove</button>
+<hr />
+</li>`;
+
+  booksListContainer.insertAdjacentHTML("beforeend", listItem);
+}
+
+form.addEventListener("submit", (e) => {
+  e.preventDefault();
+  const formData = {};
+  const title = e.target.title.value;
+  const author = e.target.author.value;
+
+  formData.title = title.trim();
+  formData.author = author.trim();
+  formData.id = `${new Date().getTime().toString()}${Math.trunc(Math.random() * 100)}`;
+
+  if (!title || !author) return;
+
+  //   Clear inputs
+  e.target.title.value = "";
+  e.target.author.value = "";
+
+  //   Add book
+  books.push(formData);
+
+  //   Save to local storage
+  //   console.log(JSON.stringify(books));
+  localStorage.setItem("books", JSON.stringify(books));
+
+  //   Append book UI
+  appendBook(formData);
+});
```

Comment:
- [ ] Kindly ensure your app is working as required. I am unable to add a book. From the screenshot below I added a book and clicked the add button but nothing happened. The book should be displayed on the page and stored in the local storage so that when the browser is refreshed the books are loaded on the page.

  ![kgitt](https://user-images.githubusercontent.com/81468675/197732410-3660236c-2301-4079-b878-bc9469bfdc7c.jpg)